README.md

```diff
@@ -62,3 +62,13 @@ A Repository to collect and showcases developers' favorite resources to learn a
     - The interactivity of the project is what helped it work for me. It is a lot easier to understand something when you're able to actually use it and see the effects.
     - For complete beginners it might be difficult to translate the lessons from this site directly to a real project, especially one where you are collaborating with other people.
 
+[Harishma.A](https://github.com/HarishmaA) 
+> Software Engineer working at Full Creative
+
+#### UDACITY
```

Comment:
We are categorizing resources based on tech-stack and not platforms. I would like you to add specific courses and tag what you like about them. For eg., people who like Java can pick it up from there

Reply:
Sure.. Sorry for late reply..

Reply:
Sure.. I will add specific courses and sorry for the late reply.